frontend/test/model/EBEventTest.scala

```diff
@@ -55,8 +57,13 @@ class EBEventTest extends PlaySpecification {
       ebDraftEvent.isBookable mustEqual(false)
     }
     "should return media service url if present" in {
-      ebLiveEvent.mainImageUrl mustEqual Some("https://some-media-tool.co.uk/images/sdf8u8sdf898hnsdcvs89dc?crop=0_3_480_288")
-
+      ebLiveEvent.mainImageUrl mustEqual Some(Uri.parse("https://some-media-tool.co.uk/images/sdf8u8sdf898hnsdcvs89dc?crop=0_3_480_288"))
+    }
+    "should tolerate additional characters after the media service url" in {
+      ebLiveEvent.copy(description = Some(EBDes)).mainImageUrl mustEqual Some(Uri.parse("https://some-media-tool.co.uk/images/sdf8u8sdf898hnsdcvs89dc?crop=0_3_480_288"))
+    }
+    "should not fail parsing if the HTML comments are malformed" in {
+      malformedHtmlEvent.mainImageUrl mustEqual Some(Uri.parse("https://some-media-tool.co.uk/images/sdf8u8sdf898hnsdcvs89dc?crop=0_3_480_288"))
```

Comment:
I think rather than parsing the malformed comment we should fail on the single event and log it?